addons/api/package.json

```diff
@@ -24,7 +24,7 @@
   },
   "dependencies": {
     "dexie": "^3.2.4",
-    "ember-auto-import": "^2.6.3",
+    "ember-auto-import": "^2.8.1",
```

Comment:
this change is mostly unrelated, saw that ember-auto-import was already resolving to newer version so decided to add the change in the package.json files to newest version.

Reply:
@lisbet-alvarez Since our upgrades are manual, I'm wondering which command you're running to determine which dependencies need updating?

Reply:
`yarn why {dep. name}` is a good resource since it tells you what version is being used and where its being used. Tho in the case of `ember-auto-import` I noticed it was resolving to a newer minor version in our `yarn.lock` file.

Reply:
I imagine you are not using `yarn why <package-name>` unless you've already determined the package needs updating, right?

Reply:
We cleared up why packages end up in [resolutions](https://github.com/hashicorp/boundary-ui/blob/main/package.json#L51) 👍 thanks!